server.js

```diff
@@ -82,71 +82,86 @@ function defaultMessageGenerator(reviewers) {
      reviewers.length > 1 ? '' : ' a',
      reviewers.length > 1 ? 's' : ''
   );
-};
+}
 
-app.post('/', function(req, res) {
-  req.pipe(bl(function(err, body) {
-    var data = {};
-    try { data = JSON.parse(body.toString()); } catch (e) {}
-
-    if (data.action !== 'opened') {
-      console.log(
-        'Skipping because action is ' + data.action + '.',
-        'We only care about opened.'
-      );
-      return res.end();
+function getRepoConfig(request): Promise<Object> {
```

Comment:
you are inconsistent in the way you name things. Sometimes you put Async at the end sometimes not. Can you pick one and use it everywhere? My preference is to remove Async suffix everywhere